ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topk/DoubleTopkKudafTest.java

```diff
@@ -21,77 +21,80 @@
 import org.junit.Before;
 import org.junit.Test;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collections;
 import java.util.List;
 
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.junit.Assert.assertThat;
 
 public class DoubleTopkKudafTest {
-  Object[] valueArray;
+  ArrayList valueArray;
   TopKAggregateFunctionFactory topKFactory;
   List<Schema> argumentType;
 
   @Before
   public void setup() {
-    valueArray = new Double[]{10.0, 30.0, 45.0, 10.0, 50.0, 60.0, 20.0, 60.0, 80.0, 35.0, 25.0};
+    valueArray = new ArrayList(Arrays.asList(10.0, 30.0, 45.0, 10.0, 50.0, 60.0, 20.0, 60.0, 80.0, 35.0,
```

Comment:
`ImmutableList.of(blah, blah)`, which suggests it can be constructed inline and the field marked `final`.